plugins/meson.build

```diff
@@ -5,10 +5,15 @@ subdir('raspberrypi')
 subdir('steelseries')
 subdir('test')
 subdir('udev')
-subdir('unifying')
 subdir('upower')
 subdir('usb')
 
+if appstream_glib.version().version_compare('>=0.6.13')
+subdir('unifying')
+else
+message('appstream-glib 0.6.13+ not found, disabling unifying plugin')
```

Comment:
What does the unifying plugin actually use the newer functionality for, out of interest? Seems a big hammer if it's only one more ifdef.